index.html

```diff
@@ -625,11 +614,15 @@ <h2>
         (depending on the underlying platform).
       </p>
       <ul>
-        <li>There is a requirement to not allow the website to learn which apps
-        are installed, or which app was chosen from {{Navigator/share()}},
-        because this information could be used for fingerprinting, as well as
+        <li>The API does not expose to the website which [=share targets=] are
+        available, or which share target was chosen by the user from
+        {{Navigator/share()}}. Doing so could be used for fingerprinting by
         leaking details about the user's device.
         </li>
+        <li>User agents are discouraged from showing any kind of "always use
+        this target in the future" option, or bypassing the UI if there is only
```

Comment:
The wording here regarding "bypassing the UI" being discouraged is not strong enough.

"prohibited" might have been unreasonably strong, but I think it better captured the intent.

Reply:
I wonder if we should flip this saying that it should always require user interaction irrespective of how many share targets are available.  

Note that we already do this normatively as part of `.share()`; this section just emphasizes what we already normatively require.

Reply:
Just to clarify, this section is just guidance. We shouldn't/can't really "prohibit" anything. 

Maybe just "strongly discourage"?

Reply:
"should always require user interaction" is fine. That makes the intent clear.

Reply:
We already normatively require "transient activation" activation to use web share (and it's verifiable for conformance). I. can state that again though as a statement of fact.